src/doc/man/cargo-report.md

```diff
@@ -0,0 +1,40 @@
+# cargo-report(1)
+
+## NAME
+
+cargo-report - Generate and display various kinds of reports
+
+## SYNOPSIS
+
+`cargo report` _type_ [_options_]
+
+### DESCRIPTION
+
+Displays a report of the given _type_ - currently, only `future-incompat` is supported
+
```

Comment:
Can this include a link to the new future-incompat chapter?  IIRC, just a relative link like `[foo](../reference/future-incompat-report.html)` should.

Reply:
I added a link in the "See also" section